pytorch_lightning/trainer/connectors/callback_connector.py

```diff
@@ -17,7 +18,8 @@ def on_trainer_init(
             process_position,
             default_root_dir,
             weights_save_path,
-            resume_from_checkpoint
+            resume_from_checkpoint,
+            differential_privacy_calllback
```

Comment:
I think it would be better to make it an explicit callback just like `LearningRateMonitor` rather than having a parameter. WDYT?